src/olympia/files/tests/test_utils_.py

```diff
@@ -272,6 +272,26 @@ def test_invalid_app_versions_are_ignored(self):
     def test_is_webextension(self):
         assert self.parse({})['is_webextension']
 
+    def test_apps_use_default_versions_if_applications_is_omitted(self):
+        """
+        WebExtensions are allowed to omit `applications[/gecko]` and we
+        previously skipped defaulting to any `AppVersion` once this is not
+        defined. That resulted in none of our plattforms being selectable.
+
+        See https://github.com/mozilla/addons-server/issues/2586 and
+        probably many others.
+        """
+        # Default AppVersions, see `ManifestJSONExtractor.app`
+        firefox_min_version = self.create_appversion('firefox', '42.0')
```

Comment:
I somehow think we should start using constants for those, e.g `MIN_WEBEXTENSION_APPVERSION` or so, any opinions on that?